src/pages/home/report/ReportActionItem.js

```diff
@@ -136,6 +133,7 @@ const defaultProps = {
 };
 
 function ReportActionItem(props) {
+    const personalDetails = usePersonalDetails() || {};
```

Comment:
I'd declare this empty object outside of the render loop

Reply:
Definitely!

Reply:
Done ✅